charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.6.7
 kubeVersion: ">=1.22.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.28.1
+version: 5.28.2
```

Comment:
Please also update the `artifacthub.io/changes: |` some lines below (L26 + L27)

Reply:
@mkilchhofer done